documentation/status-logging.md

```diff
@@ -0,0 +1,54 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging though you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
+
+    ```
+    status: {
+        logging-frequency: 2000,
+        status-logging-enabled: true
+    }
```

Comment:
@rlinehan 's suggestion does seem the most future-proof.  Maybe there's a small tradeoff in complexity but I kind of think it's worth it since it doesn't seem out of the realm of possibility that we might add another setting there in the future, e.g. a list of services to log the status from, rather than all services.

So I guess what I'm saying is I'm on board with @rlinehan 's suggestion if it seems good to you, @jpinsonault